build.sh

```diff
@@ -170,24 +175,27 @@ rm meilix-imclient_*_all.deb
 rm /sbin/initctl
 dpkg-divert --rename --remove /sbin/initctl
 
-# Unmount pseudo-filesystems within chroot
-umount -lf /proc
-umount -lf /sys
-umount -lf /dev/pts
-exit
+# Make sure we're exiting without an error code
+exit 0
```

Comment:
Hey why are we actually doing this, ignoring the errors! @yukiisbored please give an explanation according to my knowledge this is used to exit without minding the errors and trick that it was a success!